docs/proposals/extender-readiness-check.md

```diff
@@ -0,0 +1,173 @@
+# Proposal: Scheduler-extender readiness check
+
+Last updated: 08.07.2021
+
+
+## Abstract
+CSI must inform user about scheduler-extender deployment status.
+
+## Background
+#### Abbreviations
+- CM - ConfigMap with policy options 
+  (for Openshift - ConfigMap in `openshift-config` namespace used in Scheduler, for vanilla/RKE - internal configmap mounted in Patcher)
+- kube-scheduler - Pod with name `(openshift-)kube-scheduler-<node_name>`
+- extender - CSI scheduler-extender pod/daemonset
+
+#### Current implementation
+Extender will be completely installed on the system after kube-scheduler will be patched. 
+Currently, CSI provides no status if patching is completed successfully or not.
+
+Patching process on supported platforms:
+
+- For RKE/vanilla
+1. Deploy CM
+2. Deploy csi-baremetal-patcher daemonset. 
+   In each patcher:
+   - Copy police file from CM to local folder on host
+   - Update kubernetes-scheduler manifest (add path to policy into parameters)
+    
+- For Openshift
+1. Deploy CM
+2. Update schedulers.config.openshift.io CR with CM name
+
+In the both cases patching triggers kube-scheduler restart. 
+If kube-scheduler is not restarted, the extender might not work correctly.
+Restart check is implemented in E2E testing, but it's not stable.
+
+#### Patching limitations
+
+If user wants to add another scheduler-extender, he has to manage configs for CSI extender himself.
+
+Manual patching:
+- For Openshift
+1. Set `scheduler.patcher.enable = false`
```

Comment:
It's not clear where to set. Please reference to helm chart value. Don't we need use platform parameter instead?

Reply:
updated, remove `scheduler.patcher.enable` in the related PR into Operator